src/test/scala/dpla/ingestion3/enrichments/StringUtilsTest.scala

```diff
@@ -198,4 +198,41 @@ class StringUtilsTest extends FlatSpec with BeforeAndAfter {
     val enrichedValue = originalValue.findAndRemoveAll(stopWords)
     assert(enrichedValue === "photograph")
   }
+
+  "stripBrackets" should "remove leading and trailing ( )" in {
```

Comment:
@guegueng Can you look at these tests for `stripBrackets` and let me know if you think I've addressed all the possible cases here?